spring-boot/src/test/java/org/dcsa/conformance/springboot/ConformanceApplicationTest.java

```diff
@@ -84,7 +80,7 @@ private void checkUntilScenariosAreReady(String sandboxId) throws InterruptedExc
         Thread.sleep(7_000L);
       }
     } while (!status.equals("{\"scenariosLeft\":0}"));
-    assertEquals("{\"scenariosLeft\":0}", status, "Scenario did not finish. Original start status: " + startStatus);
+    assertEquals("{\"scenariosLeft\":0}", status, "Scenario '" + sandboxId + "' did not finish properly! Original start status: " + startStatus);
```

Comment:
It was confusing to see which `sandboxId` really failed. Adding this should make it very clear, I hope.